### ISSUE TITLE
Feature request: Incorporate GitHub discussions into operations

### ISSUE THREAD

**Issue by lukefretwell** [CivicActions/accessibility#250]
**Is your feature request related to a problem? Please describe.**

Build out a way to support folks who have questions about accessibility or general CivicActions Accessibility operations.

**Describe the solution you'd like**

Use GitHub Discussions (https://github.com/CivicActions/accessibility/discussions) and direct folks to it via a /help page.

```
Get CivicActions Accessibility and general accessibility support the following ways:

* Submit your question to our GitHub Discussions board
* Post your question to Slack #accessibility
``` 

**Describe alternatives you've considered**

Something similar to what we've done with DistributedGov: 

https://distributedgov.com/help

**Additional context**

cc @mgifford @dmundra for any other feedback per our conversation.

**Comment by mgifford** [comment]
Let's modify the categories from:
- General
- Ideas
- Q&A
- Show & Tell

to align with our website
- Q&A
- Playbook
- Guide

We can then point folks from the website to specific spaces to ask questions, like for general [Q&A](https://github.com/CivicActions/accessibility/discussions/categories/q-a).

**Comment by dmundra** [comment]
@lukefretwell I think can be marked done for now!
